spec/lib/rbac/filterer_spec.rb

```diff
@@ -2072,7 +2072,7 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
           grp = i + 1
           guest_os = %w(_none_ windows ubuntu windows ubuntu)[grp]
           vm = FactoryBot.build(:vm_vmware, :name => "Test Group #{grp} VM #{i}")
-          vm.hardware = FactoryBot.build(:hardware, :cpu_sockets => (grp * 2), :memory_mb => (grp * 1.megabytes), :guest_os => guest_os)
+          vm.hardware = FactoryBot.build(:hardware, :cpu_sockets => (grp * 2), :ram_size_in_bytes => (grp * 1.megabytes * 1024 * 1024), :guest_os => guest_os)
```

Comment:
think this is fine the way it was.

Not sure if we get more benefit by you fixing this (like you did), or leaving the tests unchanged (so we show that there was no major change to the code)